flash-messages/Gemfile

```diff
@@ -0,0 +1,20 @@
+source "https://rubygems.org"
+
+gem "sinatra"
+gem "data_mapper"
```

Comment:
Thoughts here: I wanted a real example of using flash to display error messages for invalid models, and notification on a successful db transaction.

Seems like a light amount of extra cruft with a higher payout in context and demonstrating Real World Examples. :D